src/test/java/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,180 @@
+
+        sa.assertAll();
+    }
+
+    // 6. Assert that there are 4 images on the Index Page and they are displayed
+    @Test
+    public void indexPageImagesTest() {
+        SoftAssert sa = new SoftAssert();
+
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
+        List<WebElement> images = driver.findElements(By.xpath("//div[@class='benefit-icon']/span"));
+
+        for (WebElement element : images) {
+            sa.assertTrue(element.isDisplayed());
+        }
+
+        sa.assertAll();
+    }
+
+    // 7. Assert that there are 4 texts on the Index Page under icons and they have proper text
+    @Test
+    public void textUnderIconsTest() {
+        SoftAssert sa = new SoftAssert();
+
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
+
+        List<WebElement> iconTexts = driver.findElements(By.className("benefit-txt"));
+        List<String> expectedIconTexts = Arrays.asList("To include good practices\n" +
+                        "and ideas from successful\n" +
+                        "EPAM project",
+                "To be flexible and\n" +
+                        "customizable", "To be multiplatform",
+                "Already have good base\n" +
+                        "(about 20 internal and\n" +
+                        "some external projects),\n" +
+                        "wish to get more…");
+        List<String> actualIconTexts = new ArrayList<>();
+
+        for (WebElement element : iconTexts) {
+            actualIconTexts.add(element.getText());
+        }
+
+        for (WebElement element : iconTexts) {
+            sa.assertTrue(element.isDisplayed());
+        }
+
+        sa.assertEquals(actualIconTexts, expectedIconTexts);
+        sa.assertAll();
+    }
+
+    // 8, 9, 10 steps
+    @Test
+    public void iframeExistsTest() {
+        SoftAssert sa = new SoftAssert();
+
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
+        WebElement iframe = driver.findElement(By.id("frame"));
+
+        // 8. Assert that there is the iframe with “Frame Button” exist
+        sa.assertTrue(iframe.isDisplayed());
+
+        driver.switchTo().frame(iframe);
+        WebElement frameButton = driver.findElement(By.id("frame-button"));
+
+        // 9. Switch to the iframe and check that there is “Frame Button” in the iframe
+        sa.assertTrue(frameButton.isDisplayed());
+
+        // 10. Switch to original window back
+        driver.switchTo().defaultContent();
+
+        sa.assertAll();
+
+    }
+
+    // 11. Assert that there are 5 items in the Left Section are displayed and they have proper text
+    @Test
+    public void leftSectionMenuTest() {
+        SoftAssert sa = new SoftAssert();
+
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
+        List<WebElement> leftSectionItems = driver.findElements(By.xpath("//ul[@class='sidebar-menu']/li"));
```

Comment:
Why do you decide use xpath here? I suppose that this locator could be changed to css and simplified